packages/garbo/src/outfit/target.ts

```diff
@@ -44,6 +44,9 @@ export function meatTargetOutfit(
   if (target === $location`Crab Island`) {
     const meat = meatDrop($monster`giant giant crab`) + songboomMeat();
     outfit.modifier.push(`${meat / 100} Meat Drop`, "-tie");
+  } else if (target === $location`Cobb's Knob Treasury`) {
```

Comment:
unfortunately I think we need a more precise check for this given that wanderers can be placed in the treasury (they can't be placed in crab island)